projects/extension/src/components/IconWeb3.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FunctionComponent } from "react"
+import "../main.css"
+
+interface Props {
+  size?: string
+  color?: string
+  children?: string
+}
+
+const hasGlyph = (string: string) =>
+  ["kusama", "polkadot", "westend", "kulupu", "rococo"].indexOf(string) > -1
```

Comment:
```suggestion
const hasGlyph = (string: string) =>
  ["kusama", "polkadot", "westend", "rococo"].indexOf(string) > -1
```

Kulupu is so random here. There are dozens of chains more important than them.

Reply:
These are the chains supported at the moment from our "font" library.